### PR TITLE
Document the outfit datum with autodoc

### PR DESCRIPTION
**Pull request by optimumtact**
Looks like this
https://www.oranges.net.nz/~oranges/dmdoc/datum/outfit.html

### CODE REVIEW

**Review by SpaceManiac — APPROVED** [review]

File: code/datums/outfit.dm [hunk 1]
```diff
@@ -1,41 +1,153 @@
...
+/**
+  * Called at the start of the equip proc
+  *
+  * Override to change the value of the slots depending on client prefs, species and
+  * other such sources of change
+  *
+  * Extra Arguments
+  * * visualsOnly true if this is only for display (in the character setup screen)
+  *
+  * If visualsOnly is true, you can omit any work that doesn't visually appear on the character sprite
```
Comment (AnturK) [inline code/datums/outfit.dm]: should -> need , that's how you get weird bugs and runtimes.

Comment (optimumtact) [reply code/datums/outfit.dm]: should does not appear in this line?

Comment (optimumtact) [reply code/datums/outfit.dm]: do you mean can -> must?

Comment (AnturK) [reply code/datums/outfit.dm]: Yes, i was just hallucinating that one.

**Pull request merged**